digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ChangeApi.scala

```diff
@@ -16,6 +16,12 @@ class ChangeApi(roadAddressService: RoadAddressService, nodesAndJunctionsService
   val logger: Logger = LoggerFactory.getLogger(getClass)
   val DateTimePropertyFormat: DateTimeFormatter = DateTimeFormat.forPattern("dd.MM.yyyy HH:mm:ss")
 
+  val XApiKeyDescription =
+    "You need an API key to use Viite APIs.\n" +
+    "Get your API key from the responsible system owner (järjestelmävastaava)."
```

Comment:
the responsible system owner ei nyt ihan tunnu oikealta käännökseltä

system administrator
https://fi.glosbe.com/fi/en/j%C3%A4rjestelm%C3%A4vastaava
https://fi.techdico.com/k%C3%A4%C3%A4nn%C3%B6s/suomi-englanti/tekninen+j%C3%A4rjestelm%C3%A4vastaava.html

Reply:
Vaihdettu  "technical ---" (muut kaks filua kans).
Joo, note: ton virallisen käännöksen voisin kyllä varmistaa Väylältä.